manifests/ntpd.pp

```diff
@@ -0,0 +1,19 @@
+# Configure NTP daemon
+class foreman_network::ntpd {
+  if ($foreman_network::configure_ntpd) {
+
+    exec { 'NTP sync':
```

Comment:
@ekohl thanks, it looks much cleaner. I used that and added path attribute to exec to avoid difficulties on some debian versions.